v22.1/api-support-policy.md

```diff
@@ -0,0 +1,108 @@
+| [`information_schema` system catalog](information-schema.html)          | TBC in review    | Versioned concurrently with CockroachDB.                                               |                                                                                                                                                              |
+| [`pg_catalog` system catalog](pg-catalog.html)                          | TBC in review    | Versioned concurrently with CockroachDB.                                               |                                                                                                                                                              |
+| [`pg_extension` system catalog](pg-extension.html)                      | TBC in review    | Versioned concurrently with CockroachDB.                                               |                                                                                                                                                              |
+| [Health Endpoints](monitoring-and-alerting.html#health-endpoints)       | Programmable     | No new versions forthcoming.                                                           |                                                                                                                                                              |
+| [Prometheus Endpoint](monitoring-and-alerting.html#prometheus-endpoint) | Programmable     | No new versions forthcoming.                                                           | Although this endpoint is not versioned, individual metrics may be added or removed in each CockroachDB release. No changes are expected to response format. |
+| [Cluster API](cluster-api.html)                                         | Programmable     | [Versioned independently from CockroachDB.](cluster-api.html#versioning-and-stability) |                                                                                                                                                              |
+| [Logging](logging-overview.html)                                        | Mixed            | Versioned concurrently with CockroachDB.                                               | Programmability varies by [event type](eventlog.html). Structured events are programmable and unstructured events are not programmable.                      |
+| [`cockroach` commands](cockroach-commands.html)                         | Non-programmable | N/A                                                                                    |                                                                                                                                                              |
```

Comment:
I don't think we want to say that all `cockroach` commands are not programmable, do we? Users will thread at least some commands into their automation, and this contradicts the rule that adding a new mandatory flag is backward-incompatible.